our_site/planner/views.py

```diff
@@ -187,6 +187,15 @@ def edit_event(request, event_id: int) -> HttpResponse:
                 event.description = posted_details.cleaned_data["event_description"]
                 event.location = posted_details.cleaned_data["event_location"]
                 event.save()
+                return HttpResponseRedirect("/planner/event_owned/{:d}/".format(event.id))
+
+    if request.method == "POST":
```

Comment:
I think something might be up here  because we already have a place where the request method is checked and the form is messed around with already before we get here. This code should probably be inside the switch statement above.